alb.tf

```diff
@@ -48,6 +48,18 @@ resource "aws_alb_listener_rule" "public" {
   listener_arn = data.aws_lb_listener.public[count.index].arn
   priority     = var.alb_listener_priority
 
+  dynamic "action" {
+    for_each = var.alb_cogino_pool_arn == null ? [/*noop*/] : ["enabled"]
```

Comment:
we could also check for empty string instead of `null`, but fine for me

Reply:
done.